### PR TITLE
shahriar048/c01-aws01

### PR DESCRIPTION
**Pull request by shahriar048**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-aws01/shahriar048/README.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+$ `aws ec2 create-security-group --group-name my-sg --description "My security group" --vpc-id vpc-49e2eb2e`
+
+Output:
+    {
+        "GroupId": "sg-0d9198b56e119769d"
+    }
+```
+
+```
+$ `aws ec2 authorize-security-group-ingress --group-id sg-0d9198b56e119769d --protocol tcp --port 22 --cidr 0.0.0.0/0`
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/shahriar048/README.md:28]: By allowing this cidr `0.0.0.0/0`, anyone in the internet can access this instance.
Would there be more secure way to expose this instance?

File: classes/01class/exercises/c01-aws01/shahriar048/README.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+    https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/
+    5 package(s) needed for security, out of 7 available
+    Run "sudo yum update" to apply all updates.
+```
+
+- Commands to create the second EC2 instance and any additional resource required:
+
+```
+$ `aws ec2 run-instances \
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/shahriar048/README.md:166]: considering a more secure security group than the one created previously can you also make a more secure SG for this private instance?
Feel free to create a different SG for this instance only

**Comment by shahriar048** [comment]
Hi Kiko, I'll address your reviews as soon as possible.

**Comment by roperto** [comment]
Hi @shahriar048  -- the deletion is automatic, you can always make a new one if it gets deleted.

However, if you are having any kind of issues submitting or getting "git" to work, please ask me on slack I'd be happy to help. Ask on #public-academy-jun-2020 feel free to tag me @roperto there

Cheers,

Daniel

**Comment by shahriar048** [comment]
Thanks, Daniel. @roperto 

I've made the changes requested by @kikobr82 and reopened the PR.

**Pull request merged**